EventFilter/RPCRawToDigi/interface/RPCTwinMuxRecord.h

```diff
@@ -0,0 +1,417 @@
+#ifndef EventFilter_RPCRawToDigi_RPCTwinMuxRecord_h
+#define EventFilter_RPCRawToDigi_RPCTwinMuxRecord_h
+
+#include <stdint.h>
+
+namespace rpctwinmux {
+
+class TwinMuxRecord
+{
+public:
+    /** Some of the types as defined in TwinMux_uROS_payload_v12.xlsx  **/
+    static unsigned int const rpc_first_type_  = 0;
+    static unsigned int const rpc_second_type_ = 1;
+    static unsigned int const error_type_      = 2;
+    static unsigned int const unknown_type_    = 3;
+
+    static ::uint64_t const twinmux_header_identifier_mask_ = 0xf000000000000000;
+    static ::uint64_t const twinmux_header_identifier_      = 0x0000000000000000;
```

Comment:
```twinmux_header_identifier_ ``` and  ```error_identifier_ ``` are apparently not used in this PR.
Are they expected to be in some upcoming version update?